tests/active_features/workflow.feature

```diff
@@ -16,20 +16,25 @@ Feature: View site workflow information
     """
 
   @vcr workflows_show
-  Scenario: Show a specific Workflow's Details and Operations
-    When This step is implemented I will test: showing a specific workflow's details
-    When I run "terminus workflow:info [[test_site_name]] 11111111-1111-1111-1111-111111111111"
-    Then I should get:
-    """
-    Deploy a CMS (Drupal or Wordpress)
-    """
-    And I should get:
-    """
-    Take Screenshot
-    """
+  Scenario: Show a specific Workflow's status
+    When I run "terminus workflow:info:status [[test_site_name]] --id=11111111-1111-1111-1111-111111111111"
+    Then I should get: "Deploy a CMS (Drupal or Wordpress)"
+    And I should get: "succeeded"
 
   @vcr workflows_show
+  Scenario: Show a specific Workflow's operations
+    When I run "terminus workflow:info:operations [[test_site_name]] --id=11111111-1111-1111-1111-111111111111"
+    Then I should get: "Apply any hostname changes"
+    And I should get: "Take Screenshot"
+
+  @vcr workflows_show
+  Scenario: Try show a Workflow that has no logs
+    When I run "terminus workflow:info:logs [[test_site_name]] --id=11111111-1111-1111-1111-111111111111"
+    Then I should see a notice message: Workflow operations did not contain any logs.
+
+  @vcr quicksilver_workflows_show.yml
   Scenario: Show the most recent set of logs for a workflow that has logs
-    When This step is implemented I will test: Show the most recent set of logs for a workflow that has logs
-    When I run "terminus workflow:info [[test_site_name]] --latest-with-logs"
-    Then I should see an error message: No recent workflow has logs
+    When This step is implemented I will test: Show the most recent workflow's logs
```

Comment:
Oh I'm glad I asked... that's a pretty good signal that the fact this represents a pending step is, well, not very representative :-) Maybe I should have written it:
`When This step is PENDING I will not try to: ...`
or
`When PENDING the rest of these steps are ignored`

Any better ideas? I was really hoping for a stand-alone annotation that might be more obvious to us all.